common/template_string.h

```diff
@@ -0,0 +1,80 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_COMMON_TEMPLATE_STRING_H_
+#define CARBON_COMMON_TEMPLATE_STRING_H_
+
+#include "llvm/ADT/StringRef.h"
+
+namespace Carbon {
+
+// Represent a compile-time string in a form suitable for non-type template
+// arguments.
```

Comment:
```suggestion
// Represents a compile-time string in a form suitable for use as a non-type
// template argument.
```
(Totally trivial but I misparsed this on first read.)